packages/react-storage/src/components/StorageBrowser/createStorageBrowser/__tests__/createStorageBrowser.spec.tsx

```diff
@@ -42,6 +43,10 @@ describe('createStorageBrowser', () => {
     });
   });
 
+  it('should not produce static type errors', async () => {
+    await expectTypeTestsToPassAsync(__filename);
+  });
```

Comment:
What does the failing case look like here? Having a hard time grokking whether this assertion belngs in this test file